manifest-list.md

```diff
@@ -64,6 +66,13 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
+- **`elements`** *array of objects*
+
+  This OPTIONAL property contains a list of non-manifest objects.
```

Comment:
I don't follow why these things can't just go into the manifests list? We already have a namespaced mediaType for all of the descriptors so adding an extension field doesn't really buy much AFAICS.